src/main/resources/application.properties

```diff
@@ -1,11 +1 @@
 
-spring.datasource.url=jdbc:h2:mem:testdb
-spring.datasource.username=sa
-spring.datasource.password=
-
-spring.datasource.driverClassName=org.h2.Driver
-spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
-
-spring.h2.console.enabled=true
-
-spring.h2.console.path=/h2
```

Comment:
add: spring.profiles.active=h2 here

Reply:
I will add it for now, but later the active profile will be different, since we talked about using postgreSQL.